packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -103,6 +105,10 @@ export const prepareVaultDirector = (
 
   // Non-durable map because param managers aren't durable.
   // In the event they're needed they can be reconstructed from contract terms and off-chain data.
+  /**
+   * @template T
+   * @type {MapStore<Brand<AssetKind>, ReturnType<makeVaultParamManager>>}
```

Comment:
what's the template for? the value needs a typeof. AssetKind is the default parameter for Brand.
```suggestion
   * @type {MapStore<Brand, ReturnType<typeof makeVaultParamManager>>}
```

Reply:
Whoops, that's a refactoring typo.